diff -ru /Users/eschnett/src/LORENE/C++/Include/nbr_spx.h LORENE/C++/Include/nbr_spx.h
--- /Users/eschnett/src/LORENE/C++/Include/nbr_spx.h	2007-06-16 17:19:23.000000000 -0500
+++ LORENE/C++/Include/nbr_spx.h	2009-11-21 17:46:49.000000000 -0600
@@ -151,6 +151,10 @@
 #include <math.h>
 #define __infinity INFINITY
 #else
+#ifdef __x86_64__
+#include <math.h>
+#define __infinity INFINITY
+#else
 extern double __infinity ;
 #endif
 #endif
@@ -158,5 +162,6 @@
 #endif
 #endif
 #endif
+#endif
 
 #endif
diff -ru /Users/eschnett/src/LORENE/Makefile LORENE/Makefile
--- /Users/eschnett/src/LORENE/Makefile	2004-03-26 08:00:21.000000000 -0600
+++ LORENE/Makefile	2009-11-21 17:43:37.000000000 -0600
@@ -29,4 +29,4 @@
 test: 
 	cd Test; $(MAKE)
 
-
+.PHONY: all install doc cpp fortran export uninstall test
