Index: Lorene/C++/Source/Bin_hor/binhor_hh.C
===================================================================
RCS file: /cvsroot/Lorene/C++/Source/Bin_hor/binhor_hh.C,v
retrieving revision 1.3
diff -u -r1.3 binhor_hh.C
--- Lorene/C++/Source/Bin_hor/binhor_hh.C	9 Jan 2008 14:28:58 -0000	1.3
+++ Lorene/C++/Source/Bin_hor/binhor_hh.C	27 Mar 2014 13:38:39 -0000
@@ -885,7 +885,7 @@
   cout << hole1.mp.phi << endl ;
   cout << hole1.mp.tet << endl ;
 
-
+  /*
   //des_meridian(hh1, 0., 20., "hh1 cart", 20) ;
   for (int i=1 ; i<= 3 ; i++)
     for (int j=i ; j<= 3 ; j++){
@@ -894,7 +894,7 @@
       //des_profile(hh1(i,j), 0., 20., 0, M_PI) ;
       des_coupe_z (hh1(i,j), 0., 5) ;
     }
-
+   */
   hh1.change_triad(hole1.mp.get_bvect_spher()) ;
   hh2.change_triad(hole2.mp.get_bvect_spher()) ;
 
@@ -908,7 +908,7 @@
   hole2.tgam = tgam_2 ;
 
   
-  des_meridian(hh1, 0., 20., "hh1", 0) ;
+  //des_meridian(hh1, 0., 20., "hh1", 0) ;
 
 
 }
Index: Lorene/C++/Source/Isol_hor/init_data.C
===================================================================
RCS file: /cvsroot/Lorene/C++/Source/Isol_hor/init_data.C,v
retrieving revision 1.28
diff -u -r1.28 init_data.C
--- Lorene/C++/Source/Isol_hor/init_data.C	19 Aug 2008 06:42:00 -0000	1.28
+++ Lorene/C++/Source/Isol_hor/init_data.C	27 Mar 2014 13:38:39 -0000
@@ -1261,13 +1261,13 @@
 
     for (int mer=0; mer<niter; mer++) {
 
-      
+      /*
       des_meridian(psi_j, 1, 10., "psi", 0) ;
       des_meridian(nn_j, 1, 10., "nn", 1) ;
       des_meridian(theta_j, 1, 10., "Theta", 2) ;
       des_meridian(chi_j, 1, 10., "chi", 3) ;
       arrete() ;
-
+      */
 
       //========
       // Sources
@@ -1408,11 +1408,11 @@
 	   
 	 // Iteration in theta/chi
 	 for (int i=0 ; i<niter ; i++) {
-
+      /*
 	   des_meridian(theta_i, 1, 10., "Theta", 2) ;
 	   des_meridian(chi_i, 1, 10., "chi", 3) ;
 	   arrete() ;
-
+      */
 
 
 	   //Sources
Index: Lorene/C++/Source/Time_slice/tslice_conf_init.C
===================================================================
RCS file: /cvsroot/Lorene/C++/Source/Time_slice/tslice_conf_init.C,v
retrieving revision 1.11
diff -u -r1.11 tslice_conf_init.C
--- Lorene/C++/Source/Time_slice/tslice_conf_init.C	20 Oct 2010 07:58:09 -0000	1.11
+++ Lorene/C++/Source/Time_slice/tslice_conf_init.C	27 Mar 2014 13:38:39 -0000
@@ -225,7 +225,7 @@
         maxabs(psi_jp1.laplacian() - source_psi,
                 "Absolute error in the resolution of the equation for Psi") ;  
 
-        des_meridian(psi_jp1, 0., ray_des, "Psi", ngraph0, graph_device) ; 
+        //des_meridian(psi_jp1, 0., ray_des, "Psi", ngraph0, graph_device) ; 
 
         // Resolution of the Poisson equation for the lapse
         // ------------------------------------------------
@@ -238,21 +238,21 @@
         maxabs(nn_jp1.laplacian() - source_nn,
                 "Absolute error in the resolution of the equation for N") ;  
 
-        des_meridian(nn_jp1, 0., ray_des, "N", ngraph0+1, graph_device) ; 
+        //des_meridian(nn_jp1, 0., ray_des, "N", ngraph0+1, graph_device) ; 
         
         // Resolution of the vector Poisson equation for the shift
         //---------------------------------------------------------
         
         Vector beta_jp1 = source_beta.poisson(0.3333333333333333, ff, 
                                               method_poisson_vect) ; 
-        
+        /*
         des_meridian(beta_jp1(1), 0., ray_des, "\\gb\\ur\\d", ngraph0+2, 
                      graph_device) ; 
         des_meridian(beta_jp1(2), 0., ray_des, "\\gb\\u\\gh\\d", ngraph0+3, 
                      graph_device) ; 
         des_meridian(beta_jp1(3), 0., ray_des, "\\gb\\u\\gf\\d", ngraph0+4, 
                      graph_device) ; 
-        
+        */ 
         // Test:
         Vector test_beta = (beta_jp1.derive_con(ff)).divergence(ff)
             +  0.3333333333333333 * (beta_jp1.divergence(ff)).derive_con(ff) ;
Index: Lorene/C++/Source/Time_slice/tslice_dirac_max_evolve.C
===================================================================
RCS file: /cvsroot/Lorene/C++/Source/Time_slice/tslice_dirac_max_evolve.C,v
retrieving revision 1.20
diff -u -r1.20 tslice_dirac_max_evolve.C
--- Lorene/C++/Source/Time_slice/tslice_dirac_max_evolve.C	24 Jan 2013 12:55:18 -0000	1.20
+++ Lorene/C++/Source/Time_slice/tslice_dirac_max_evolve.C	27 Mar 2014 13:38:39 -0000
@@ -688,7 +688,7 @@
             beta_new = relax * beta_new + (1.-relax) * beta() ;
 	    beta_evol.update(beta_new, jtime, ttime) ;             
 	}    
-                
+   /*        
         des_meridian(vec_X()(1), 0., ray_des, "\\gb\\ur\\d", ngraph0+6,
                      graph_device) ; 
         des_meridian(vec_X()(2), 0., ray_des, "\\gb\\u\\gh\\d", ngraph0+7,
@@ -713,6 +713,7 @@
                      graph_device) ; 
                 
         arrete(nopause) ; 
+    */
     }
 
     par_A.clean_all() ;
