diff -ur Lorene.orig/C++/Include/nbr_spx.h Lorene/C++/Include/nbr_spx.h
--- Lorene.orig/C++/Include/nbr_spx.h	2009-11-22 11:06:11.000000000 -0600
+++ Lorene/C++/Include/nbr_spx.h	2021-09-09 21:07:04.000000000 -0500
@@ -158,7 +158,12 @@
 #include <math.h>
 #define __infinity INFINITY
 #else
-extern double __infinity ;
+#ifdef __arm64__
+#include <math.h>
+#define __infinity INFINITY
+#else
+extern double __infinity;
+#endif
 #endif
 #endif
 #endif
